css/civicrmNavigation.css

```diff
@@ -91,6 +91,7 @@ li.menu-separator.active{
   display:block;
   font-weight:normal;
   text-decoration:none;
+  border-bottom-style:none;
```

Comment:
Since this is an `li`, I'd probably use `border-style: initial;` (which defaults to "none", but to me it reads more sensibly to say "default behaviour" than specify a particular behaviour).

This feels trivial but @seamuslee001 asked me to comment so ... was that your question SL?